### PR TITLE
=cls #19917 Clarify docs about sharding ddata

### PR DESCRIPTION
**Pull request by patriknw**
Refs #19917

### CODE REVIEW

**Comment by johanandren** [comment]
LGTM

File: akka-docs/rst/dev/index.rst [hunk 1]
```diff
@@ -9,5 +9,4 @@ Information for Akka Developers
    io-layer
    developer-guidelines
    documentation
-   team
```
Comment (ktoso) [inline akka-docs/rst/dev/index.rst]: Thanks, must have missed it

File: akka-docs/rst/java/cluster-sharding.rst [hunk 2]
```diff
@@ -184,11 +184,18 @@ Distributed Data Mode
...
 as storage for the state of the sharding coordinator. In such case the state of the 
-``ShardCoordinator`` will be replicated inside a cluster by the Distributed Data module with
+``ShardCoordinator`` will be replicated inside a cluster by the :ref:`distributed_data_java` module with
 ``WriteMajority``/``ReadMajority`` consistency.
 
-This mode can be enabled by setting configuration property ``akka.cluster.sharding.state-store-mode``
-as ``ddata``. 
+This mode can be enabled by setting configuration property::
+
+    akka.cluster.sharding.state-store-mode=ddata
```
Comment (ktoso) [inline akka-docs/rst/java/cluster-sharding.rst]: nitpick: I'd personally like spaces around the setting, makes it easier to parse where key ends.

**Comment by ktoso** [comment]
LGTM

File: akka-docs/rst/java/cluster-sharding.rst (hunk 2 above)
Comment (patriknw) [inline akka-docs/rst/java/cluster-sharding.rst]: me too, will change

**Pull request merged**